src/core/utils.js

```diff
@@ -153,3 +158,21 @@ export function browserBase64Decode(str) {
     return '%' + ('00' + c.charCodeAt(0).toString(16)).slice(-2);
   }).join(''));
 }
+
+export function apiAddonType(addonType) {
+  if (!Object.prototype.hasOwnProperty.call(
```

Comment:
Oh, crazy eslint rules. Probably makes sense but shucks is that ugly. Would saving `hasOwnProperty` off work? I can't decide if it's better to be explicit or try and make it look a little nicer. Either way works for me.

```js
const hasOwnProperty = Object.prototype.hasOwnProperty.call;

export function apiAddonType(addonType) {
  if (!hasOwnProperty(API_ADDON_TYPES_MAPPING, addonType)) { }
  /* ... */
}